pkg/api/norman/customization/kontainerdriver/actionhandler.go

```diff
@@ -81,20 +80,8 @@ func (a ActionHandler) refresh(apiContext *types.APIContext) error {
 		return httperror.WrapAPIError(err, httperror.ServerError, msg)
 	}
 
-	setting, err := a.MetadataHandler.SettingLister.Get("", rkeMetadataConfig)
-	if err != nil {
-		return err
-	}
-
-	if setting.Annotations == nil {
-		setting.Annotations = make(map[string]string)
-	}
-
-	setting.Annotations[forceRefreshAnnotation] = strconv.FormatInt(time.Now().Unix(), 10)
```

Comment:
`forceRefreshAnnotation` is no longer used anywhere

Reply:
Do you know how/why it was used before? Is this a problem in 2.5?

Reply:
It was used earlier to refresh k3s metadata based on this annotation, the logic still exists in 2.5 but no longer in 2.6 https://github.com/rancher/rancher/blob/release/v2.5/pkg/controllers/managementapi/k3smetadata/settings_handler.go#L66